### PR TITLE
Fix videoplayback on NC15

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

**Comment by rullzer** [comment]
@MorrisJobke can you test safari
@juliushaertl can you test IE11 and edge?

File: js/viewer.js [hunk 1]
```diff
@@ -5,10 +5,20 @@ var videoViewer = {
...
 			} else {
 				this.videoJSLoaded = true;
-				var stylePath = OC.filePath('files_videoplayer', 'videojs', 'src/video-js.css');
+				var stylePath = OC.filePath('files_videoplayer', 'js', 'videojs/video-js.css');
 				$('head').append($('<link rel="stylesheet" type="text/css" href="' + stylePath + '"/>'));
-				var scriptPath = OC.filePath('files_videoplayer', 'videojs', 'src/video.js');
-				return $.getScript(scriptPath, function (xhr) {eval(xhr);});
+				var scriptPath = OC.filePath('files_videoplayer', 'js', 'videojs/video.js');
+
+				var deferred = $.Deferred();
```
Comment (ChristophWurst) [inline js/viewer.js]: this is duplicating https://github.com/nextcloud/server/blob/b7767a51f140d3f588278f053f3a9e1eb84e4346/core/js/js.js#L364-L385, isn't it? Does `$.getScript` load the script differently?

Comment (rullzer) [reply js/viewer.js]: Yes. I have no idea why or how but it is different. With `$.getScript` I did not have a `videojs` funtion after loading. This way i do.

Comment (rullzer) [reply js/viewer.js]: @ChristophWurst but it probably would make sense to look into a generic approach here indeed. This is just the quick fix for 15.

Comment (ChristophWurst) [reply js/viewer.js]: I see, they are indeed a bit different. Let's use this fix for 15 now and look into a proper npm setup for 16 where we can use dependabot and video.js from https://www.npmjs.com/package/video.js.

Comment (skjnldsv) [reply js/viewer.js]: We could integrate that with the slideshow/big picture we discussed in https://github.com/nextcloud/server/issues/12382

Comment (ChristophWurst) [reply js/viewer.js]: Yeah, sounds good. So let's npm'ize this repo for 16, switch to Vue and use a standardized component for the overlay.

**Comment by jancborchardt** [comment]
Thanks a lot for fixing this @rullzer! :tada:

**Pull request merged**